tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -29,46 +29,55 @@
 #ifndef TF2_EIGEN_H
 #define TF2_EIGEN_H
 
+#include <chrono>
+
 #include <tf2/convert.h>
 #include <Eigen/Geometry>
-#include <geometry_msgs/PointStamped.h>
-#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/msg/point_stamped.hpp>
+#include <geometry_msgs/msg/pose_stamped.hpp>
 
 
 namespace tf2
 {
-    
-Eigen::Affine3d transformToEigen(const geometry_msgs::TransformStamped& t) {
+
+Eigen::Affine3d transformToEigen(const geometry_msgs::msg::TransformStamped& t) {
   return Eigen::Affine3d(Eigen::Translation3d(t.transform.translation.x, t.transform.translation.y, t.transform.translation.z)
-			 * Eigen::Quaterniond(t.transform.rotation.w, 
+			 * Eigen::Quaterniond(t.transform.rotation.w,
 					      t.transform.rotation.x, t.transform.rotation.y, t.transform.rotation.z));
 }
 
 
 // this method needs to be implemented by client library developers
 template <>
-void doTransform(const tf2::Stamped<Eigen::Vector3d>& t_in, 
+void doTransform(const tf2::Stamped<Eigen::Vector3d>& t_in,
 		 tf2::Stamped<Eigen::Vector3d>& t_out,
-		 const geometry_msgs::TransformStamped& transform) {
+		 const geometry_msgs::msg::TransformStamped& transform) {
+  std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds> converted{std::chrono::nanoseconds{transform.header.stamp.sec*1000000000LL+transform.header.stamp.nanosec}};
+  std::chrono::system_clock::time_point recovered = std::chrono::time_point_cast<std::chrono::system_clock::duration>(converted);
   t_out = tf2::Stamped<Eigen::Vector3d>(transformToEigen(transform) * t_in,
```

Comment:
I'm not quite sure I understand; there are 2 spaces after the opening function bracket everywhere in this file, so I just followed that convention.  What specifically do you mean?